CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## [Unreleased]
 
+## [2.0.0] - 2017-05-02
+### Added
+- Added new `tslint` rules from [v5.0.0](https://github.com/palantir/tslint/blob/master/CHANGELOG.md#v500) and [v5.1.0](https://github.com/palantir/tslint/blob/master/CHANGELOG.md#v510) releases
```

Comment:
Ahhhh, I assumed you'd like to have each rules value listed. 

I like the `(disabled)` suggestion. 👍